src/Parsers/ASTColumnsTransformers.cpp

```diff
@@ -61,23 +64,42 @@ void ASTColumnsExceptTransformer::formatImpl(const FormatSettings & settings, Fo
 
 void ASTColumnsExceptTransformer::transform(ASTs & nodes) const
 {
+    ASTs unexcepted_columns(children);
```

Comment:
typo :   actually should be named as `expected_columns`

Reply:
expected_column or excepted_column?